### DISCUSSION TITLE
Undo possible?

### DISCUSSION THREAD

**Discussion by LiamKrenn** [pocketbase/pocketbase#3391] [Q&A] [answered]
Is there a way to undo the last change?

**Comment by ganigeorgiev** [comment, marked as answer]
If by _"undo the last change"_ you mean restore the previous data of a single record, then No (at least not out of the box) because we don't keep history of the record db changes.

If by _"undo the last change"_ you mean restore previous collection configuration, then if automigrate is enabled (which is the default with the prebuilt executable) you can check the generated migration file in your `pb_migrations` and either edit the migration or run `./pocketbase migrate down` and delete the related migration file(s).
